### PR TITLE
Changed README

### PR DESCRIPTION
**Pull request by FriedChckn**
Just customized it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# TotalFreedomMod #
+# Wolfcraft Freedom #
 
-TotalFreedomMod is a CraftBukkit server plugin designed primarily to support the [Official TotalFreedom Minecraft Server](http://totalfreedom.me/). However, you are more than welcome to adapt the source for your own server.
+WolfcraftFreedomMod is a CraftBukkit server plugin designed primarily to support the [Official TotalFreedom Minecraft Server](http://totalfreedom.me/). However, you are more than welcome to adapt the source for your own server.
```
Comment (rovertdude) [inline README.md]: This is the WolfFreedom-Core - not WolfcraftFreedomMod

**Comment by rovertdude** [comment]
Change it from WolfCraftFreedomMod to WolfFreedom-Core - other than that, ready to merge.

**Comment by rovertdude** [comment]
Merging right now  modifying it when it's merged.

**Comment by FriedChckn** [comment]
Im ready to upload to the server

**Comment by rovertdude** [comment]
@FriedChckn upload it!

**Pull request merged**